src/tools/crawling_tools.py

```
@@ -348,6 +474,103 @@ async def crawl(ctx: Context, url: str, max_depth: int = 3, max_concurrent: int
+                        node_type=page_type,
+                        node_id=doc.url,
+                        properties={
+                            "url": doc.url,
+                            "domain": domain,
+                            "title": markdown_content.split('\n')[0][:100] if markdown_content else "",
+                            "content_length": len(markdown_content),
+                            "crawled_at": time.time(),
+                            "page_type": page_type,
+                            "section_count": len(sections),
+                            "code_block_count": len(code_blocks),
+                            "api_mentions": mentioned_apis[:10]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)